install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -52,7 +52,27 @@ type WorkspaceConfig struct {
 }
 
 type WebAppConfig struct {
-	PublicAPI *PublicAPIConfig `json:"publicApi,omitempty"`
+	PublicAPI         *PublicAPIConfig `json:"publicApi,omitempty"`
+	WorkspaceDefaults struct {
```

Comment:
Should we move these under a specific component, like `Server`? Feels like we can end up with a lot of config and it may be unclear whether it belongs to all, or just some components.

Reply:
Related, I think comments for what each value is meant to be could be helpful here, following the Go style guide conventions. For example, I wouldn't immediately know what `Session` is and what it's used for.

Reply:
> Should we move these under a specific component, like `Server`? Feels like we can end up with a lot of config and it may be unclear whether it belongs to all, or just some components.

Moved in 6e3cd0347f53ee0c9c71e25435862cbbeee45cd0 (at the cost of some more `nil` checks).

Reply:
> Related, I think comments for what each value is meant to be could be helpful here, following the Go style guide conventions. For example, I wouldn't immediately know what `Session` is and what it's used for.

The problem is that I'm not sure how all of the config values are used; that would require a deeper investigation how `server` consumes this config. @geropl and I are working off a diff of what the installer produces vs the old helm chart and adding configuration switches to the installer where we need to.

Reply:
Thanks. I think the nil check is largely due to our usage of experimental at this stage. It should go away when this becomes the mainline way of deploying and will have values as part of the regular config.